src/app/zap_cluster_list.json

```diff
@@ -154,7 +155,9 @@
         "APPLICATION_LAUNCHER_CLUSTER": ["application-launcher-server"],
         "AUDIO_OUTPUT_CLUSTER": ["audio-output-server"],
         "BALLAST_CONFIGURATION_CLUSTER": [],
+        "BARRIER_CONTROL_CLUSTER": ["barrier-control-server"],
```

Comment:
Why is this being added?  It shouldn't be here.

Reply:
Apologies, I don't know what was happening, alchemy has been freaking out on me - I was synced@head, not sure where this came from

Started over & looks good now